source/_integrations/knx.markdown

```diff
@@ -212,7 +212,20 @@ type:
 
 ### Read
 
-You can also use the `homeassistant.update_entity` service call to issue GroupValueRead requests for all `*state_address` of a device.
+You can use the `homeassistant.update_entity` service call to issue GroupValueRead requests for all `*state_address` of an entity.
+To manually send GroupValueRead requests use the `knx.read` service. The response can be used from `knx_event` and will be processed in KNX entities.
```

Comment:
I can understand this is quite hard for people to follow. Maybe add a practical automation example?

Reply:
I've added an example for the service.

Reply:
Yes, but triggering the service, is not the interesting part in this. How one can process the data returned from the event, is interesting as well. So, can we extend the example to entail this?

Reply:
KNX is a local push service. Under normal circumstances manually reading group addresses is not required, but there are some edge cases where it can matter. It also heavily depends on the actuator linked to the group address.

I can't really come up with a useful example that can be understandable at a glance with processing data from the knx_event. If the group address is configured in an Entity this entity will update, but I can't really show this in an automation.
So I made an example that reads a group address representing a cover position after a configurable time. Some actuators don't automatically send these positions, so the example actually has practical use case 😀